src/app/social/look/page.jsx

```
@@ -0,0 +1,593 @@
+            { key: 'recent', label: '🕐 Most Recent', icon: '🕐' },
+            { key: 'popular', label: '❤️ Most Liked', icon: '❤️' },
+            { key: 'trending', label: '🔥 Trending', icon: '🔥' }
+          ].map((option) => (
+            <button
+              key={option.key}
+              onClick={() => setSortBy(option.key)}
+              style={{
+                padding: '10px 20px',
+                borderRadius: '15px',
+                border: 'none',
+                background: sortBy === option.key 
```

Biome (1.9.4)

[error] This property value named border is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named border


(lint/suspicious/noDuplicateObjectKeys)